hopsworks-common/pom.xml

```diff
@@ -373,7 +373,13 @@
       <version>3.9.1</version>
       <scope>provided</scope>
     </dependency>
-    
+
+    <dependency>
+      <groupId>com.logicalclocks</groupId>
+      <artifactId>service-discovery-client</artifactId>
```

Comment:
Does it bring any risky transient dependencies, such as jackson or jersey?

Reply:
The dependencies are shaded https://github.com/logicalclocks/service-discovery-clients/blob/master/java/pom.xml#L76